### ISSUE TITLE
3.5.0 Typings not updated

### ISSUE THREAD

**Issue by jrakotoharisoa** [angular-redux/ng-redux#153]
Angular module name is no more exported as default.

So this code no more working
``` typescript
import NgRedux from 'ng-redux';

angular.module('app', [NgRedux]);
```

Now we should do that, but typing  doesn't allow it:
``` typescript
import * as NgRedux from 'ng-redux';

angular.module('app', [NgRedux]);
```

**Comment by Nagogus** [comment]
Have you found any way to import 3.5.0? For me the only solution was to downgrade to 3.4.1.

**Comment by jrakotoharisoa** [comment]
Temporarily, I use : 

```typescript
const NgRedux = require('ng-redux');

angular.module('app', [NgRedux]);
```

**Comment by jrakotoharisoa** [comment]
Fix with #155

**Comment by AntJanus** [comment]
The fix for this will be released in version 3.5.1
